### ISSUE TITLE
Added Image Rotate

### ISSUE THREAD

**Issue by andrewcampey** [kensnyder/quill-image-resize-module#48]
Not an issue but more of a tip.

I added Image rotate to this library and it was pretty simple.
Changes:-
- Added buttons to the current align tool bar
- Added name so I can track when rotate buttons are clicked (They can be clicked repeatedly)
- Added basic function to check where the image was in the rotation cycle.
- Used the quill Undo and Redo buttons as my buttons.

It only requires the Toolbar.js file to be modified

**Comment by accgetter** [comment]
@andrewcampey 
please add. I need this code!

**Comment by andrewcampey** [comment]
You can find my code here:-
https://github.com/andrewcampey/quill-image-resize-module

Or its on npm (https://www.npmjs.com/package/quill-image-resize-and-rotate-module)

However there are still bugs that need fixing.
1) When you rotate the image it doesn't recenter the image inside the Quill control
![image](https://user-images.githubusercontent.com/2282361/42740527-1bcf4e4e-88ec-11e8-94d1-3292b4264861.png)

2) The alignment and rotating doesn't appear to fire off a change event to the Quill editor. To get around this currently I have the following:-
					```
	var width = this.img.width;
	this.img.width = width - 1;
	this.img.width = width + 1;
```
This works to fire the change event however adds one to the width so need a better option less hacky option.

Don't have time to investigate currently

**Comment by benwinding** [comment]
@andrewcampey your package has the buttons, but the images don't rotate when the button is clicked. Can you confirm if this is working?

**Comment by benwinding** [comment]
For anyone looking for the solution, I've published a package `quill-image-resize-module` which works well alongside this package `quill-image-resize-module`, as shown below:

![image](https://user-images.githubusercontent.com/11782590/68917960-520c3800-07bc-11ea-9832-78a0f83c5efa.png)

Github: https://github.com/benwinding/quill-image-rotate-module#readme
Npm: https://www.npmjs.com/package/quill-image-rotate-module

**Comment by andrewcampey** [comment]
Nice work Ben. It SHOULD have worked.

The biggest issue is on large images it would appear outside the box. have
you had this issue?

**Comment by benwinding** [comment]
Yes, there's some issues with using rotate() in css which need to be fixed with custom margins. The rotate module works well by itself.

Try here: https://benwinding.github.io/quill-image-rotate-module/demo/index.html

I didn't realise that @kensnyder had made functionality to include your own modules here: https://github.com/kensnyder/quill-image-resize-module#basemodule---include-your-own-custom-module

That might have been a better solution, as the `quill-image-resize-module` isn't aware of the image resizing.
